script/zip.js

```diff
@@ -1,14 +1,42 @@
+import { writeFileSync, unlinkSync, mkdirSync, rmdirSync } from "node:fs";
 import { Zip } from "zip-lib";
 
 import manifest from "../manifest.json" assert { type: "json" };
+const backgroundFile = "lib/background.js";
 
-const zip = new Zip();
+const manifestForVendors = {
```

Comment:
@schalkneethling Please let me know if this conflicts as well 😊

Reply:
@schalkneethling Please let me know if this conflicts as well 😊